e2e/condition_test.go

```diff
@@ -0,0 +1,55 @@
+// +build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/consul-terraform-sync/testutils"
+	ctsTestClient "github.com/hashicorp/consul-terraform-sync/testutils/cts"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCondition_CatalogServices_Include runs the CTS binary. It specifically
+// tests a task configured with a catalog service condition with the
+// source_includes_var = true. This test confirms that the catalog_service
+// definition can be consumed by a module as expected.
+func TestCondition_CatalogServices_Include(t *testing.T) {
+	srv := newTestConsulServer(t)
+	defer srv.Stop()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "cs_condition_include")
+	delete := testutils.MakeTempDir(t, tempDir)
+
+	conditionTask := `task {
+	name = "catalog_task"
+	services = ["api"]
+	source = "../../test_modules/local_tags_file"
+	condition "catalog-services" {
+		regexp = "db|web"
+		source_includes_var = true
```

Comment:
Aha! I missed this on my first reading. The source_includes_var is set in the section for the conditional variable, not a more global setting as I imagined. This makes more sense to me know.

(sorry community member, we talked about this in slack but basically I was confused about how this was used)

Reply:
Gotcha! Ok, I'm glad to hear that this cleared it up! I considered adding an example config in the PR description but decided against it, darn. Seems like it would have been helpful for contextualizing 'source_includes_var'. I'll add it anyways after the fact. Thanks for the feedback!